handlers/group.ts

```diff
@@ -0,0 +1,62 @@
+import {randomUUID} from 'expo-crypto';
+interface IGroup {
+    create(groupName: string, action : () => void): void;
+    join(uuid: string, action : (ack: Boolean) => void): void;
+    checkIfInGroup(action : (ack: Boolean) => void): void ;
+    leave() : Boolean;
+    setPeers() : void;
+}
+
+class GroupHandle implements IGroup {
+
+    constructor(private gun: Gun) {        
+    }
+
+    public create(groupName: string, action : () => void): void {
+        let user = gun.user(userPub)
+        let groupId : string = randomUUID();        
+        let context = gun.get("groups").set({groupId: groupId})
+        context.get("members").set({members: userPub})
+        context.get("name").set({groupName: groupName})        
```

Comment:
This seems weird, first of all name should not be a set, so use put here, and won't it create an object that looks like {groupId: groupId, name: {groupName: groupName}}
Would expect either `context.put({groupName: groupName})` or `context.get("groupName").put(groupName)`